.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: rust
 cache:
 - cargo
+before_install:
+- MAX_LINE_LENGTH=80 bash ci/check_line_lengths.sh src/**/*.md
```

Comment:
This seems to only be running on the src/test/ directory. Perhaps this should be changed to

```
- MAX_LINE_LENGTH=80 bash ci/check_line_lengths.sh src/*.md src/**/*.md 
```

Reply:
Yeah, actually, `src/**.md` may be what we want...?

Reply:
Nope, it looks like that doesn't get the tests/ directory.